tools/topology/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ add_custom_target(abi
 set(TPLGS
 	"sof-cht-nocodec\;sof-cht-nocodec\;-DPLATFORM=cht"
 	"sof-cht-nocodec\;sof-byt-nocodec\;-DPLATFORM=byt"
+	"sof-hsw-nocodec\;sof-hsw-nocodec\;-DPLATFORM=hsw"
```

Comment:
we don't support haswell, why bother?

Reply:
OK, will remove. So do we need to remove the build for HSW FW as well?

Reply:
yes